dist/index.html

```diff
@@ -21,6 +28,6 @@
       You need to enable JavaScript to run this app.
     </noscript>
 
-    <script src="./bundle.js"></script>
+    <script src="../dist/bundle.js"></script>
```

Comment:
Don't think this change is necessary here since the file itself is in the `dist` folder.

Reply:
makes sense